### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Vayts**
https://Vayts.github.io/layout_moyo-header/
https://Vayts.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,43 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
```
Comment (Moroz-Dmytro) [inline src/index.html:10]: Import only font-family and font-weight that you are using

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,43 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="logo">
```
Comment (Moroz-Dmytro) [inline src/index.html:16]: Do we really need so many nested divs?

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,43 @@
...
+              <img
+                src="../src/images/logo.png"
+                alt="logo"
+                class="logo__img"
+              >
+            </a>
+          </div>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
```
Comment (Moroz-Dmytro) [inline src/index.html:27]:
```suggestion
              <li class="nav__item">
                <a href="#apple" class="nav__link is-active">
                  Apple
                </a>
              </li>

```

Comment (Moroz-Dmytro) [reply src/index.html:27]: Please, fix this in all places

File: src/index.html [hunk 4]
```diff
@@ -7,8 +7,43 @@
...
+              <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+              <li class="nav__item"><a href="#smarthphones" class="nav__link">Smartphones</a></li>
+              <li class="nav__item">
+                <a
+                data-qa="hover"
+                href="#laptopscomputers"
+                class="nav__link"
+                >
```
Comment (Moroz-Dmytro) [inline src/index.html:35]:
```suggestion
                <a
                  data-qa="hover"
                  href="#laptopscomputers"
                  class="nav__link"
                >
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,80 @@
...
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  justify-content: space-around;
+}
+
+.nav__link.is-active {
```
Comment (Moroz-Dmytro) [inline src/style.css:49]: image
And in all other places

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,80 @@
...
+
+.header__content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.container {
+  max-width: 1200px;
+  min-width: 1024px;
```
Comment (Moroz-Dmytro) [inline src/style.css:32]: what about smaller screens?
image

**Review by Moroz-Dmytro — CHANGES_REQUESTED** [review]
Good job 👍 
Left some comments
Also, when you create PR, please add links to demo and test results to PR description

File: src/index.html [hunk 7]
```diff
@@ -7,42 +7,52 @@
...
-          <nav class="nav">
-            <ul class="nav__list">
-              <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
-              <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
-              <li class="nav__item"><a href="#smarthphones" class="nav__link">Smartphones</a></li>
-              <li class="nav__item">
-                <a
+      <div class="header__content container">
+        <a href="#home" class="logo">
+          <img
```
Comment (alexander-ignatow) [inline src/index.html]: Still some redundant nested `div` - please address @Moroz-Dmytro 's comment!

File: src/style.css [hunk 8]
```diff
@@ -46,13 +42,25 @@ body {
...
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  font-size: 12px;
+  margin-left: 20px;
+}
+
+.is-active {
   color: var(--color__nav);
   position: relative;
```
Comment (alexander-ignatow) [inline src/style.css:59]: You are altering `position` for active links

In your case it is a bad idea - I would suggest making the same `position` for all `nav_link`s as changing
* gives browser additional unnecessary job
* may break your layout one day
* may look confusing for other developers

File: src/index.html (hunk 7 above)
Comment (alexander-ignatow) [inline src/index.html]: Still some redundant nested div - please address @Moroz-Dmytro 's comment!

**Review by alexander-ignatow — CHANGES_REQUESTED** [review]
Great - most issues are fixed 💪 

Lets make final effort!

**Review by BudnikOleksii — APPROVED** [review]
Good job👍🏻

**Pull request closed without merging**